src/Microsoft.Azure.SignalR.Common/ServiceConnections/StrongServiceConnectionContainer.cs

```diff
@@ -35,8 +35,17 @@ public override Task HandlePingAsync(PingMessage pingMessage)
 
         public override Task StopAsync()
         {
-            var stopOndemandConnectionsTask =  Task.WhenAll(_onDemandServiceConnections.Select(c => c.StopAsync()));
-            return Task.WhenAll(stopOndemandConnectionsTask, base.StopAsync());
+            var tasksForOndemandConnections = Task.WhenAll(_onDemandServiceConnections.Select(c => c.StopAsync()));
+            return Task.WhenAll(tasksForOndemandConnections, base.StopAsync());
+        }
+
+        public override Task ShutdownAsync(TimeSpan timeout)
+        {
+            _shutdown = true;
```

Comment:
>_shutdown [](start = 12, length = 9)

only base class to set this, maybe for base class to `virtual ShutdownCore` instead